lib/routes/evenings.js

```diff
@@ -14,4 +14,28 @@ module.exports = router
                 .then(evening => res.json(evening))
                 .catch(next);
         });
+    })
+
+    .put('/:id', ensureAuth, (req, res, next) => {
+        const { id } = req.params;
+        const { rating } = req.body;
+        Evening.findOneAndUpdate(
+            { _id: id },
+            { rating: rating },
+            { new: true, runValidators: true })
+            .populate({ path: 'logs', select: { __v: false } })
+            .select({ __v: false })
```

Comment:
you dont need this because `toJSON` does this.

Reply:
we need it because of `lean()`